app/ante/ante.go

```diff
@@ -1,67 +1,89 @@
 package ante
 
 import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/crypto/keys/ed25519"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	"github.com/cosmos/cosmos-sdk/crypto/types/multisig"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
-	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
 	authante "github.com/cosmos/cosmos-sdk/x/auth/ante"
-	"github.com/cosmos/cosmos-sdk/x/auth/keeper"
-	"github.com/cosmos/cosmos-sdk/x/auth/types"
+	authsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
 
 	"github.com/cosmos/ethermint/crypto/ethsecp256k1"
 	evmtypes "github.com/cosmos/ethermint/x/evm/types"
-
-	tmcrypto "github.com/tendermint/tendermint/crypto"
 )
 
-func init() {
-	ethsecp256k1.RegisterCodec(types.ModuleCdc)
-}
-
 const (
 	// TODO: Use this cost per byte through parameter or overriding NewConsumeGasForTxSizeDecorator
 	// which currently defaults at 10, if intended
 	// memoCostPerByte     sdk.Gas = 3
 	secp256k1VerifyCost uint64 = 21000
 )
 
+// AccountKeeper defines an expected keeper interface for the auth module's AccountKeeper
+type AccountKeeper interface {
+	authante.AccountKeeper
```

Comment:
does this need to conform to the sdk type AccountKeeper? or is this ethermint's custom AccountKeeper interface?

Reply:
looks like the previous declaration was of this type on `/Users/danielchoi/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.39.2/x/auth/keeper/keeper.go` line 19-29

Reply:
An interface is better to us  only the required functions for the ante handler

Reply:
sorry i dont quite understand what that means